test/generators/ModelGenerators.scala

```diff
@@ -246,4 +247,46 @@ trait ModelGenerators {
         DisplaySubscriptionDetails(requestCommon, requestDetail)
       )
     }
+
+  implicit val arbitraryCreateSubscriptionForCBCResponseCommon: Arbitrary[ResponseCommon] =
+    Arbitrary {
+      for {
+        status         <- arbitrary[String]
+        processingDate <- arbitrary[LocalDateTime]
+      } yield ResponseCommon(status, processingDate)
+    }
+
+  implicit val arbitraryCreateSubscriptionForCBCResponseDetail: Arbitrary[ResponseDetail] =
+    Arbitrary(validSubscriptionID.map(ResponseDetail.apply))
+
+  implicit val arbitraryCreateSubscriptionForCBCResponse: Arbitrary[CreateSubscriptionForCBCResponse] =
+    Arbitrary {
+      for {
+        responseDetail <- arbitrary[ResponseDetail]
+        responseCommon <- arbitrary[ResponseCommon]
+      } yield CreateSubscriptionForCBCResponse(responseCommon, responseDetail)
+    }
+
+  implicit val arbitraryCreateSubscriptionResponse: Arbitrary[CreateSubscriptionResponse] =
+    Arbitrary(arbitrary[CreateSubscriptionForCBCResponse].map(CreateSubscriptionResponse.apply))
+
+  implicit val arbitrarySubscriptionAuditDetails: Arbitrary[SubscriptionAuditDetails] =
+    Arbitrary {
+      for {
+        request  <- arbitrary[CreateSubscriptionForCBCRequest]
+        response <- arbitrary[CreateSubscriptionForCBCResponse]
+      } yield SubscriptionAuditDetails(
```

Comment:
Can't we use `SubscriptionAuditDetails.fromSubscriptionRequestAndResponse(...)` here?